src/systems/actions.cairo

```diff
@@ -16,21 +16,6 @@ mod actions {
     use starknet::{ContractAddress, get_caller_address};
     use dojo_starter::models::{position::{Position, Vec2}, moves::{Moves, Direction}};
 
-    // // declaring custom event struct
```

Comment:
i think we should keep starknet::Event & add a dojo::Event example
and show different use cases
i've added a PlayerMood ex in doc https://book.dojoengine.org/cairo/events#custom-events

Reply:
Ok. I will revert it as it was before, fixed on commit 198d4d7. At the end I added #[dojo::event] as an extra, is ok like this?

Reply:
yes, so example using dojo-starter should keep working, and just need an update to add example with dojo::Event